practice.kt

```diff
@@ -0,0 +1,24 @@
+
+    
+fun main() {
+    ////////task 2 30-40 mint
+    var list = listOf<String>("brainXTechnologies-brainXTechnologies","brainXTechnologies","Hello", "World", "in", "a", "frame" )
```

Comment:
make this list as parameter of the function.

Reply:
done